pandas/tests/series/test_analytics.py

```diff
@@ -1641,6 +1641,37 @@ def test_value_counts_categorical_not_ordered(self):
         tm.assert_series_equal(s.value_counts(normalize=True), exp)
         tm.assert_series_equal(idx.value_counts(normalize=True), exp)
 
+    @pytest.mark.parametrize("func", [np.any, np.all])
```

Comment:
@gfyoung are these tests for uses of `nv.validate*` collected anywhere?  I couldn't find where they were.

Reply:
No, we do not.  Leaving them organized as they are (in their respective function tests) is fine.